code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -1,3 +1,6 @@
+//For use in prob(), about a 1 in 121 chance
+#define WAND_WREST_CHANCE 0.82644628
```

Comment:
```suggestion
#define WAND_WREST_CHANCE (1/121)
```

Reply:
I'll change it (100/121) because for some reason byond has the input for prob() be a percentage and not a decimal.